input/pagecontent/StructureDefinition-fi-scheduling-service-event-intro.md

```diff
@@ -0,0 +1 @@
+### Scope and Usage
```

Comment:
Are you planning to add any content here? I believe it should
- refer to the base profiles (https://hl7.fi/fhir/finnish-base-profiles/StructureDefinition-fi-base-encounter.html#relation-to-finnish-palvelutapahtuma)
- state that "when the service event id (*palvelutapahtuman tunniste*) is included into the Appointment resource, it SHALL be done with this extension
- potentially explain that there are other mechanisms for communicating this information too, like including it into the Provenance, as Kanta does, or referring to the Appointment resource from the Encounter.

Reply:
Added some content.